[2025-04-23T12:16:00.368+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-04-23T12:16:00.380+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: preprocessing_pipeline.run_preprocessing manual__2025-04-23T12:15:57.732118+00:00 [queued]>
[2025-04-23T12:16:00.386+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: preprocessing_pipeline.run_preprocessing manual__2025-04-23T12:15:57.732118+00:00 [queued]>
[2025-04-23T12:16:00.386+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-04-23T12:16:00.392+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_preprocessing> on 2025-04-23 12:15:57.732118+00:00
[2025-04-23T12:16:00.398+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=241) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-23T12:16:00.399+0000] {standard_task_runner.py:63} INFO - Started process 243 to run task
[2025-04-23T12:16:00.400+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'preprocessing_pipeline', 'run_preprocessing', 'manual__2025-04-23T12:15:57.732118+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/dag_preprocessing.py', '--cfg-path', '/tmp/tmpajsnk5i2']
[2025-04-23T12:16:00.401+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask run_preprocessing
[2025-04-23T12:16:00.421+0000] {task_command.py:426} INFO - Running <TaskInstance: preprocessing_pipeline.run_preprocessing manual__2025-04-23T12:15:57.732118+00:00 [running]> on host 558735b76edf
[2025-04-23T12:16:00.464+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='preprocessing_pipeline' AIRFLOW_CTX_TASK_ID='run_preprocessing' AIRFLOW_CTX_EXECUTION_DATE='2025-04-23T12:15:57.732118+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-23T12:15:57.732118+00:00'
[2025-04-23T12:16:00.465+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-04-23T12:16:00.465+0000] {dag_preprocessing.py:92} INFO - Esecuzione del task di preprocessing iniziata.
[2025-04-23T12:16:00.466+0000] {dag_preprocessing.py:27} INFO - Avvio del preprocessing dei dati...
[2025-04-23T12:16:00.466+0000] {dag_preprocessing.py:31} INFO - Percorso previsto per il CSV: /opt/***/sentiment_data/all-data.csv
[2025-04-23T12:16:00.466+0000] {dag_preprocessing.py:34} INFO - Caricamento del file CSV...
[2025-04-23T12:16:00.467+0000] {dag_preprocessing.py:38} ERROR - File non trovato: /opt/***/sentiment_data/all-data.csv. Verifica che il file sia stato copiato nel container e che il path sia corretto.
[2025-04-23T12:16:00.467+0000] {dag_preprocessing.py:97} ERROR - Errore nel task di preprocessing: [Errno 2] No such file or directory: '/opt/***/sentiment_data/all-data.csv'
[2025-04-23T12:16:00.467+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-04-23T12:16:00.467+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-04-23T12:16:00.471+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=preprocessing_pipeline, task_id=run_preprocessing, run_id=manual__2025-04-23T12:15:57.732118+00:00, execution_date=20250423T121557, start_date=20250423T121600, end_date=20250423T121600
[2025-04-23T12:16:00.501+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-04-23T12:16:00.510+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-23T12:16:00.511+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
